rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive and evaluate
+applications to the team or alter the composition at any time with approval
+from the NixOS Foundation board. The team's composition and announcements
+should be maintained at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities for the [community
+discussions](https://nixos.org/community/index.html) on behalf of the NixOS
```

Comment:
Presumably for the official spaces?

Reply:
That's the implication. But I'd guess that the team can decide to expand or limit how/where they do moderation. For example, perhaps they leave moderation in some specific spaces up to the hosting provider's mechanisms. Or they may offer their help to unofficial spaces in a limited capacity. I don't know how it will all play out and didn't want to tie them down too much.

Reply:
It's a pretty unclear implication when you link to a list of a ton of spaces. I think that avoiding having an official space where nobody recognised by th ecommunity can moderate is desirable in any case, so fine to make an assumption in this direction. Maybe something along the lines of «perform moderation activities for the discussions in the official project spaces, as well as unofficial spaces that seek and reach such an agreement with the team.»?